apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -193,17 +265,72 @@ class MoveStudents extends Component {
         />
         <BaseDialog
           useUpdatedStyles
-          isOpen={this.state.isDialogOpen}
+          isOpen={isDialogOpen}
           style={styles.dialog}
           handleClose={this.closeDialog}
         >
-          <Table.Provider
-            columns={columns}
-            style={styles.table}
-          >
-            <Table.Header />
-            <Table.Body rows={sortedRows} rowKey="id" />
-          </Table.Provider>
+          <div style={styles.container}>
+            <Table.Provider
+              columns={columns}
+              style={styles.table}
+            >
+              <Table.Header />
+              <Table.Body rows={sortedRows} rowKey="id" />
+            </Table.Provider>
+            <div style={styles.rightColumn}>
+              <div>{i18n.selectStudentsToMove()}</div>
+              <label
+                htmlFor="sections"
+                style={styles.label}
+              >
+                {`${i18n.moveToSection()}:`}
+              </label>
+              <select
+                name="sections"
+                style={styles.input}
+                onChange={this.onChangeSection}
+              >
+                {this.renderOptions()}
+              </select>
+              {otherTeacherSelected &&
+                <div>
+                  <label
+                    htmlFor="sectionCode"
+                    style={styles.label}
+                  >
+                    {`${i18n.enterSectionCode()}:`}
+                  </label>
+                  <input
+                    required
+                    name="sectionCode"
+                    style={{...styles.input, width: 225}}
+                    value={otherTeacherSectionValue}
+                    onChange={this.onChangeTeacherSection}
+                    placeholder={i18n.sectionCodePlaceholder()}
+                  />
+                  <label style={styles.label}>{i18n.bothSectionsQuestion()}</label>
+                  <label style={styles.input}>
+                    <input
+                      type="radio"
+                      value={COPY}
+                      checked={copyStudents}
+                      onChange={this.onChangeMoveOrCopy}
+                    />
+                    <span style={styles.radioOption}>{i18n.copyStudentsConfirm()}</span>
+                  </label>
+                  <label style={styles.input}>
+                    <input
+                      type="radio"
+                      value="move"
+                      checked={!copyStudents}
+                      onChange={this.onChangeMoveOrCopy}
+                    />
+                    <span style={styles.radioOption}>{i18n.moveStudentsConfirm()}</span>
+                  </label>
+                </div>
+              }
+            </div>
+          </div>
```

Comment:
no need to do this right now, but you'll probably want to extract a subcomponent or two as this component grows.

Reply:
yeah, i was thinking about that. this component is definitely at its limit right now... i feel like there's a few places it could be broken up. maybe i'll pull the 'other teacher' functionality into its own component

Reply:
yeah, the "other teacher" stuff seems like a great candidate for extraction.